src/demandlib/particular_profiles.py

```diff
@@ -85,7 +171,14 @@ def simple_profile(self, annual_demand, **kwargs):
         self.dataframe.loc[night_filter & weekend_filter, "ind"] = (
             profile_factors["weekend"]["night"]
         )
+        self.dataframe.loc[day_filter & holiday_filter, "ind"] = (
+            profile_factors["holiday"]["day"]
+        )
+        self.dataframe.loc[night_filter & holiday_filter, "ind"] = (
+            profile_factors["holiday"]["night"]
+        )
 
+        # Check for NAN values in the dataframe
         if self.dataframe["ind"].isnull().any(axis=0):
             logging.error("NAN value found in industrial load profile")
```

Comment:
Why would that happen?

Reply:
This check has been there before. It can happen when weekdays, weekend days and holidays are set inconsistently, for example when day 5 does not appear in any of the lists for "week", "weekend" or "holiday". I can't think of any other constellation leading to NaN values, but maybe it can also happen due to something else.

Reply:
I see. Maybe just don't add the comment. As you added it, I assumed that you were knowledgeable about possible errors. In fact, the error code is not included to the unit tests.

PS: What happens can be seen in the code. Comments should tell, why something is done or useful. So, this comment is not needed anyway.

Reply:
Building on your comment I wanted to add a test for the NaN error raising but realised, that NaN values are not really possible, as we initialise the dataframe column "ind" with zeros. I can't think of a case that could lead to NaN values, so we could also take out the entire NaN check. What do you think?

Reply:
I would have suggested the same but did not really know if there might be cases where NAN may occur. If its hardly possible, please delete the check.

Reply:
Thank you for the quick replies :) I took out the NaN check.